nilearn/signal.py

```diff
@@ -477,22 +518,36 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
                 session_confounds = confounds[sessions == s]
             signals[sessions == s, :] = \
                 clean(signals[sessions == s],
-                      detrend=detrend, standardize=standardize,
+                      detrend=detrend, normalize=normalize,
                       confounds=session_confounds, low_pass=low_pass,
                       high_pass=high_pass, t_r=2.5)
 
-    # detrend
+    # Override normalize by deprecated standardize
+    if standardize:
+        normalize = 'std'
+        warnings.simplefilter('always', DeprecationWarning)
```

Comment:
I addressed your comments and fixed forgotten `normalize` in the initiation of some masker relying objects